jupyterhub/tests/test_spawner.py

```diff
@@ -56,6 +56,7 @@ def new_spawner(db, **kwargs):
     kwargs.setdefault('term_timeout', 1)
     kwargs.setdefault('kill_timeout', 1)
     kwargs.setdefault('poll_interval', 1)
+    kwargs.setdefault('port', 5555)
```

Comment:
Since starting a new spawner will generally not be passed a port, we shouldn't need a port here. If this is required for tests to pass, we may need to think about what's changing in a breaking way.